scripts/ldap/setup_ldap.sh

```diff
@@ -98,22 +99,28 @@ cp -f $INSTALLDIR/olcDatabase\=\{2\}mdb.ldif /etc/openldap/slapd.d/cn\=config/
 
 chgrp ldap /etc/openldap/certs/password # onlyif: grep -q ldap /etc/group && test -f /etc/openldap/certs/password
 
-if [[ $defaultpasswd == true ]]
-then # Get password from cortx-utils
-    cipherkey=$(s3cipher generate_key --const_key openldap)
-
-    sgiamadminpassd=$(s3confstore "$confstore_config_url" getkey --key "openldap>sgiam>secret")
-    rootdnpasswd=$(s3confstore "$confstore_config_url" getkey --key "openldap>root>secret")
-
-    # decrypt the passwords read from the confstore
-    LDAPADMINPASS=$(s3cipher decrypt --data "$sgiamadminpassd" --key "$cipherkey")
-    ROOTDNPASSWORD=$(s3cipher decrypt --data "$rootdnpasswd" --key "$cipherkey")
-else # Fetch Root DN & IAM admin passwords from User
-    echo -en "\nEnter Password for LDAP rootDN: "
-    read -s ROOTDNPASSWORD && [[ -z $ROOTDNPASSWORD ]] && echo 'Password can not be null.' && exit 1
+if [ -z "$LDAPADMINPASS" ]
+then
+    if [[ $defaultpasswd == true ]]
+    then # Use same default password for both for empty ones
```

Comment:
please improve the text of the comment, its not easy to understand what the comment is trying to convey !